src/main/java/com/arpnetworking/clusteraggregator/Main.java

```diff
@@ -58,14 +59,18 @@
      * @param args command line arguments
      */
     public static void main(final String[] args) {
+        final AtomicReference<DynamicConfiguration> configRef = new AtomicReference<>();
+        final AtomicReference<Configurator<?, ?>> configuratorRef = new AtomicReference<>();
         LOGGER.info()
                 .setMessage("Launching cluster-aggregator")
                 .log();
 
         Thread.setDefaultUncaughtExceptionHandler(
                 (thread, throwable) -> {
-                    System.err.println("Unhandled exception! exception: " + throwable.toString());
-                    throwable.printStackTrace(System.err);
+                    LOGGER.error()
```

Comment:
Are you sure this is safe to convert to using a logger? I believe we had problems before where the logger was exploding and we needed to capture the exception. Although this is after the first use of the logger, so it does seem odd.